### DISCUSSION TITLE
Gitops flux aks compliance stuck in pending

### DISCUSSION THREAD

**Discussion by thulasidassrinivasan** [Azure/AKS#3064] [Q&A] [answered]
Hi Team

I have configured gitops flux in AKS by using az cli and when i look at the compliance table some are compliant but some are stuck at pending. I am not able to find out why they are in pending status.

`az k8s-configuration flux list --resource-group $rgname --cluster-name $clustername --cluster-type $type -o table`

`Namespace       Name                   Scope      ProvisioningState    ComplianceState    StatusUpdatedAt                   SourceUpdatedAt
--------------  ---------------------  ---------  -------------------  -----------------  --------------------------------  -------------------------
cert-manager    cert-manager           cluster    Succeeded            Compliant          2022-07-06T10:44:37.619000+00:00  2022-07-06T10:44:26+00:00
**vault           hashicorp-vault        cluster    Succeeded            Pending            2022-07-06T10:51:50.163000+00:00  2022-07-06T10:31:25+00:00**
podidentity     podidentity            cluster    Succeeded            Compliant          2022-07-06T10:46:50.053000+00:00  2022-07-06T10:26:21+00:00`

Could someone help please ?

command used to enable it below

`vaultnamespace='vault'
vname='hashicorp-vault'
az k8s-configuration flux create -g $rgname -c $clustername -n $vname --namespace $vaultnamespace -t $type \
 --scope cluster -u $giturl \
 --branch $branch \
 --https-user $username \
 --https-key $password \
 --kustomization name=$vname path=./infra/vault retry-interval=10 timeout=0h10m0s`

**Comment by thulasidassrinivasan** [comment, marked as answer]
it took a awful long time before it turned from pending to complaint

**Reply by iponnam** [reply]
Did it not timeout within 10 minutes ?

**Comment by meta456trader** [comment]
I tried to use gitlab for flux configuration repo in AKS Gitops, and it stuck after I created new configuration in Azure Portal. It's okay for me. 
The problem is that I can't delete configuration. It stuck for more than 12 hours now after I press delete button.

I stopped all vmss instances, but it also didn't help. Also I couldn't delete microsoft.flux extension.
Is there any way that I can delete this configuration? 
And If I leave this configuration, how this will consume credit? Is it neglectable?
